Core/src/Geometry/Extent.cpp

```diff
@@ -104,6 +104,17 @@ void Acts::Extent::setEnvelope(const ExtentEnvelope& envelope) {
   m_envelope = envelope;
 }
 
+bool Acts::Extent::contains(const Vector3& vtx) const {
+  Extent checkExtent;
+  for (int ibv = 0; ibv < (int)binValues; ++ibv) {
+    if (constrains((BinningValue)ibv)) {
```

Comment:
Is this a c-style cast to ```BinningValue```? Is is possible to do that with ```static_cast``` or smth like that?

Reply:
Ha ha, that was co-pilot writing it, you are right, let's static cast.

Reply:
Does it not work?